tests/Tokenizer/TokensAnalyzerTest.php

```diff
@@ -68,16 +68,51 @@ class Foo2
         $tokensAnalyzer = new TokensAnalyzer($tokens);
         $elements = array_values($tokensAnalyzer->getClassyElements());
 
-        $this->assertCount(9, $elements);
-        $this->assertSame('property', $elements[0]['type']);
-        $this->assertSame('property', $elements[1]['type']);
-        $this->assertSame('property', $elements[2]['type']);
-        $this->assertSame('property', $elements[3]['type']);
-        $this->assertSame('const', $elements[4]['type']);
-        $this->assertSame('method', $elements[5]['type']);
-        $this->assertSame('method', $elements[6]['type']);
-        $this->assertSame('method', $elements[7]['type']);
-        $this->assertSame('const', $elements[8]['type']);
+        $this->assertCount(11, $elements);
+        $this->assertSame('class', $elements[0]['type'], '@{0}');
+        $this->assertSame('property', $elements[1]['type'], '@{1}');
+        $this->assertSame('property', $elements[2]['type'], '@{2}');
+        $this->assertSame('property', $elements[3]['type'], '@{3}');
+        $this->assertSame('property', $elements[4]['type'], '@{4}');
+        $this->assertSame('const', $elements[5]['type'], '@{5}');
+        $this->assertSame('method', $elements[6]['type'], '@{6}');
+        $this->assertSame('method', $elements[7]['type'], '@{7}');
+        $this->assertSame('method', $elements[8]['type'], '@{8}');
+        $this->assertSame('class', $elements[9]['type'], '@{9}');
+        $this->assertSame('const', $elements[10]['type'], '@{10}');
```

Comment:
ah, so this is actually raw-printed as literally `@{10}` ? not a template or sth...